deploy-apps/rolling-deploy.html.md.erb

```diff
@@ -173,6 +173,19 @@ The following table describes the limitations when using these deployments.
       Retrieve the logs by running the cf CLI command <code>cf logs APP_NAME</code>.
     </td>
   </tr>
+  <tr>
+    <td>Evaluating the canary instance option 2</td>
+    <td>
+      This method only works with routing-release XYZ or higher.
+      <br>
+      <br>
+      Get the canary process guid via `cf curl /v3/deployments?status_values=ACTIVE?app_guid=$(cf app APP_NAME --guid)`.
+      Add the `X-CF-PROCESS-INSTANCE:<CANARY_PROCESS_GUID>` header to traffic
```

Comment:
@ameowlia , these markdown code tags won't work in the HTML, unfortunately. You'll have to use HTML code tags.

Reply:
dang it! I always forget! ty @anita-flegg .